scripts/setup-services.sh

```diff
@@ -83,15 +83,17 @@ cd services
 echo "Setting up login.webmaker.org"
 cd login.webmaker.org
 cp env.sample .env
-sudo npm install --no-bin-links --loglevel=error # sudo needed for bcrypt permissions
+# Refer to https://github.com/npm/npm/issues/10776#issuecomment-209638410 on why we run npm install twice
+sudo npm install --no-bin-links --loglevel=error || sudo npm install --no-bin-links --loglevel=error # sudo needed for bcrypt permissions
```

Comment:
@Pomax the "double npm install needed" is still an issue with node 6 unfortunately. Although, I think instead of two, I will move it to a looped npm install.